### PR TITLE
Technological Neutrality Writeup

### PR DESCRIPTION
**Pull request by AnirudhKhemani**
@twang 

Why technology isn't neutral.

### CODE REVIEW

File: fall.txt [hunk 1]
```diff
@@ -13,3 +13,5 @@ Do not change the file extension. Save it as a .txt file
 
 
 BEGIN WRITING BELOW
+
+	At first glance, it seems plausible that technology is amoral, its negative or positive outcomes a consequence of how it is employed. One may argue, a tool's effects are the results of the actions of its wielder, and the tool itself is not responsible for its consequences. However, looking deeper, the tool has its own independent effects, whether used in one way or another.  A simple example is, a tool takes up space, impacting things around it. Furthermore, technology shapes people, as it is a part of their environment.   For example, technology can distort perception by amplifying parts of an experience. As Neil Postman states, "to a man with a pencil, everything looks like a list. To a man with a camera, everything looks like an image. To a man with a computer, everything looks like data" (Postman 1993, p. 14). Additionally, technology has biases caused by the creator. Another point is that technology can insist on being used in only certain ways, influencing the user. All in all, technology is not a neutral thing.
```
Comment (twang) [inline fall.txt]: Can you think of an example of technology that insists on being used in only certain ways?

**Comment by clairejs13** [comment]
Are there any benefits (refutations) of technology? If so, what are they?

**Comment by clairejs13** [comment]
Are there any benefits (refutations) of technology? If so, what are they?

**Comment by twang** [comment]
@clairejs13: I think we actually wanted to comment on the work in progress https://github.com/twangcs/unit-1-work-in-progress-AnirudhKhemani/pull/1, but don't worry about it now

**Pull request closed without merging**